src/osd/OSDMap.cc

```diff
@@ -3156,7 +3168,7 @@ void OSDMap::encode(ceph::buffer::list& bl, uint64_t features) const
   {
     // NOTE: any new encoding dependencies must be reflected by
     // SIGNIFICANT_FEATURES
-    uint8_t target_v = 9; // when bumping this, be aware of range blocklist
+    uint8_t target_v = 9; // when bumping this, be aware of allow_crimson
```

Comment:
Shouldn't we set `target_v` to 12? Same for `DECODE_START`? (e.g 2be95af8211225c43d72e439b6000a25743bdab7)

Reply:
No.  The idea here is that we avoid setting the more restrictive version if allow_crimson isn't set.  Just below, if allow_crimson is set, we bump target_v to 12.